cpu/stm32f3/include/periph_cpu.h

```diff
@@ -78,6 +78,35 @@ typedef enum {
     GPIO_AF15               /**< use alternate function 14 */
 } gpio_af_t;
 
+/**
+ * @brief   DAC line configuration support
+ */
+typedef struct {
+    gpio_t pin;             /**< pin connected to the line */
+#ifdef DAC2
+    uint8_t dac;            /**< The DAC to use, 0 or 1 */
+#endif
+    uint8_t chan;           /**< DAC device used for this line */
+} dac_conf_t;
+
+/**
+ * @brief   A CPU specific DAC line generator macro
+ * @{
+ */
+#ifdef DAC2
+#define DAC_LINE_CONF(port, pin, dac, chan)     { GPIO_PIN(port, pin), dac, chan }
+#else
+#define DAC_LINE_CONF(port, pin, chan)          { GPIO_PIN(port, pin), chan }
+#endif
+/** @} */
+
+/**
+ * @brief   Configure the given pin to be used as ADC input
+ *
+ * @param[in] pin       pin to configure
+ */
+void gpio_init_analog(gpio_t pin);
```

Comment:
Sinds `gpio_t` is not yet defined it can't be moved here. I'll move `gpio_t` `UNDEF` and `GPIO_PIN` to common aswell.